importers.md

```diff
@@ -121,89 +121,105 @@ Imagens enviadas por FTP,myfolder/image.jpg||myfolder/image2.jpg
+#### Automatic Creation of Metadata
 
-Quando o usuário mapeia as colunas encontradas no arquivo .csv aos metadados presentes na coleção de destino, é possível selecionar a opção `Criar Metadado`, então o importador irá criar o metadado automaticamente durante o processamento do arquivo *.csv*.
+When you map the columns found in the *.csv* file to the metadata present in the target collection, you can select the `Create Metadata` option. Then the importer will create the metadata automatically during the *.csv* file processing.
 
-Por padrão, o metadado criado será do tipo Texto e público, mas é possível informar ao Tainacan o tipo e outras opções do metadado no começo do arquivo *.csv*.
+By default, the metadata created will be of type `Text` and `public`. If you want otherwise, you can define the type and other options of the metadata at the beginning of the *.csv* file. 
 
-Na primeira linha, onde você define o nome de cada coluna, é possível adicionar algumas informações que serão usadas pelo importador para criar o `metadatum_id`.
+In the first line, where you define the name of each column, you can add some information that will be used by the importer to create the `metadatum_id`. 
 
-Cada informação sobre o metadado deve ser separada pelo caractere "`|`".
+Each information about the metadata must be separated by the character "`|`".
 
-A primeira informação deve ser o *nome do metadado*, e na sequência o *tipo do metadado*,
+The first information is the *name of the metadata*, and then follows *the type of metadata*.
 
-Os tipos de metadados suportados nativamente atualmente são:
+The types of metadata currently supported natively are:
 
-* `text` - Texto
-* `textarea` - Texto Longo
-* `taxonomy` - Taxonomia: quando esse tipo é usado, uma nova taxonomia será criada.
-* `date` - Data: os valores devem ser informados no formato YYYY-MM-DD (2018-01-01).
-* `numeric` - Numérico
-* `relationship` - Relacionamento: os valores devem ser a ID do item relacionado.
+* `text`
+* `textarea`
+* `taxonomy` - when that type is used, a new taxonomy will be created.
+* `date` - the values must be entered in the format YYYY-MM-DD (2018-01-01).
+* `numeric`
+* `relationship` - the values must be the related item ID.
 
-Por exemplo:
+Example:
 
 ```
-Nome,Assunto|taxonomy,Data de criação|date
+Name,Subject|taxonomy,Date of creation|date
 ```
 
-#### Informações sobre o metadado de Taxonomia
+#### Taxonomy metadata information
 
-Se uma das colunas do seu CSV tem valores para um **metadado de taxonomia** e esta taxonomia possui hierarquia, você pode informar essa hierarquia utilizando o sinal de `>>`.
 
-Por exemplo:
+If one of your CSV columns has values for a taxonomy metadata and this taxonomy has a hierarchy, you can enter that hierarchy using the `>>` sign.
+
+Example:
 
 ```
-nome, Categoria
-Nome do item,Categoria Pai>>Categoria Filha>>Categoria Neta
+name, category
+name of item,parent category>>child category>>needed category
```

Comment:
Here and in the line 169, I believe it its "grandchild category"  instead of "needed category"